recipes/flot/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "flot" %}
+{% set version = "0.8.3" %}
+{% set sha256 = "3a3666d0aab777b8fbfddd45146966297e6ef18777d9fd535353be1cc62b21f1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flot/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
```

Comment:
Can you please fix this as in the rest of the PRs?

Reply:
Thanks for the review. I have removed nodejs from run requirements, and added it to test/requirements